apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -21,13 +21,28 @@ the deployed code.
 
 ## Initializing Google Diagnostics (ASP.NET Core 2.0+)
 
-The preferred way to initialize Google Diagnostics services in ASP.NET Core 2.0 or higher
+The easiest way to initialize Google Diagnostics services in ASP.NET Core 2.0 or higher
 is using the `UseGoogleDiagnostics` extentension method on `IWebHostBuilder`. This configures
 Logging, Tracing and Error Reporting middleware.
 
+If your application is runnng on GAE you don't need to provide a value for `ProjectID`, `Service`
+and `Version` since this can be automatically obtained by the `UseGoogleDiagnostics` method, they
+will be the ones associated with the running application.
+
+If your application is running on GKE/GCE you don't need to provide a value for `ProjectID`
```

Comment:
"GKE/GCE" => "GKE or GCE" perhaps? (The earlier one where all three are involved is simpler with slashes.)

Reply:
Done